distribution/openrct2.d.ts

```diff
@@ -602,6 +602,24 @@ declare global {
         transparent?: boolean;
     }
 
+    type SoundOptions = SoundOptions2d | SoundOptions3d;
+
+    interface SoundOptionsBase {
+        soundId: number; // see SoundId in openrct2/audio/audio.h
```

Comment:
Instead of ids we should probably use sound names and a string union, akin to what was attempted [here](https://github.com/OpenRCT2/OpenRCT2/pull/24147/commits/430058e38f7c172b2c0b72e89317d7c9e9f12ad0#diff-db021a04c0ef4fc08087ab30a638db11919be30c71943006df4d5950bb31ade2), as the ids can shift around and break things if new sounds get added.

Reply:
@synkc Do you plan on finishing this? I made similar functionality only to find out this pr exists too, but is sort of in limbo

Reply:
I had some personal stuff going on and didn't manage the PR, I saw it too late, sorry @MaxMallon

> Instead of ids we should probably use sound names and a string union, similar to what was attempted [here](https://github.com/OpenRCT2/OpenRCT2/pull/24147/commits/430058e38f7c172b2c0b72e89317d7c9e9f12ad0#diff-db021a04c0ef4fc08087ab30a638db11919be30c71943006df4d5950bb31ade2), as the ids can shift around and break things if new sounds get added.

I'll fix it and mention it in regards to your suggestion, thanks!